### DISCUSSION TITLE
Add Points in MF34 and MF35 Sampling

### DISCUSSION THREAD

**Discussion by Huang-Yihan** [luca-fiorito-11/sandy#198] [Q&A]
I noticed that SANDY adds the incident energy points when the angular or energy distributions were perturbated. Why SANDY adds these points?  
In MF34, If the first incident energy point, e1, has 3 Legendre order coefficients, and the next point e2 has 6 Legendre order coefficients, and I noticed that the new point e3 between e1 and e2 has only  3 Legende order coefficients. Why it is not 6 Legendre order. How do you make sure your code is right?
Thank you very much.

**Comment by luca-fiorito-11** [comment]
Hi @Huang-Yihan , can you give us an example of a endf6 file with which you are working? 
Also, to clarify things from the start, do you work on branch `develop` or `master`?

**Reply by Huang-Yihan** [reply]
@luca-fiorito-11 Thank you very much. I work in the master branch. The endf-6 file is U-238 in JEFF-3.3 library. 
I recovered your results in your JEFF-3.3 Covariance Paper successfully, and  I still have the above question about adding points.

**Reply by luca-fiorito-11** [reply]
I'll try and have a look asap. Don't hesitate to contact me again in case I don't come back to you.

**Comment by luca-fiorito-11** [comment]
@AitorBengoechea , FYI
